feat(content): Burthen Triathlon culture conversation (#9455)
endless-sky/endless-sky: data/human/culture conversations.txt

			label end
			`	You say your goodbyes as you board your ships. Although you are both captains, the trajectory of your lives have taken very different courses.`
				decline

mission "Burthen Triathlon"
	minor
	source
		planet "Burthen"
	to offer
		random < 1
	on offer
		conversation

> 			`Burthen's planetary spaceport is not known to be well-trafficked because of the planet's high gravity, but today it seems to have a lot of visitors. Many of the usually empty landing pads are now occupied by shuttlecraft, and as you step out onto the spaceport's rather untarnished apron, you notice there's a sizable crowd at the edge of the spaceport, some of them obviously from offworld judging by their height. There's even a few media crews standing by, although it looks like a couple of their camera drones are struggling in Burthen's gravity.`

Reviewer: ```suggestion
			`Burthen's planetary spaceport is not a common destination thanks to the planet's high gravity, but today it seems to have a lot of visitors. Many of the usually empty landing pads are occupied by shuttlecraft and you notice there's a sizable crowd at the edge of the spaceport as you step out onto its rather untarnished apron. Some members of the crowd are obviously from offworld judging by their height, and there's even a few media crews standing by, although it looks like a couple of their camera drones are struggling in Burthen's gravity.`
```

Well-trafficked implies drugs or similar. The second sentence is also a bit of a grammatical mess.

			choice
				`	(See what's going on.)`
				`	(Avoid the commotion and tend to your own business.)`
					decline

> 			`	Every step takes significantly more effort than you're used to, but you eventually reach the crowd. From the signs that have been put up on the way it looks like the 'Burthen Triathlon' is taking place today, and the spaceport has been designated the finishing point. A couple of especially sturdy-looking marquees have been set up; the smaller one appears to be the administration tent, currently only accessible to event staff, while the other one is far larger and is catering food and drinks for the crowd nearby.`

Reviewer: ```suggestion
			`	Every step takes significantly more effort than you're used to, but you eventually reach the crowd. From the signs that have been put up on the way it looks like the "Burthen Triathlon" is taking place today, and the spaceport has been designated the finishing point. A couple of especially sturdy-looking marquees have been set up; the smaller one appears to be the administration tent, currently only accessible to event staff, while the other one is far larger and is catering food and drinks for the crowd nearby.`
```

			choice
				`	(Walk over to the catering tent.)`
				`	(Stay outside.)`
					goto outside
				`	(Return to your ship.)`
					decline

> 			`	Thankfully the walk to the catering tent is far shorter than the walk from your ship, but nevertheless you're exhausted by the time you drag your feet inside. A lot of effort has been put into this tent: the metal chairs are all sturdy to allow offworlders not used to the gravity to pull themselves up, but cushioned as to not make it uncomfortable to sit in. A small firepit has been built in the middle of the tent, along with a couple fire extinguishers placed nearby, and the whole place is brightly lit in contrast to Burthen's typically chilly weather. Quite a few people, both Burthensiders and offworlders, are mingling about here.`

Author: I don't think "keep the audience" works.
```suggestion
			`	Thankfully, the walk to the catering tent is far shorter than the walk from your ship, but nevertheless you're exhausted by the time you drag your feet inside. A lot of effort has been put into this tent: the metal chairs are all sturdily-built to allow offworlders to pull themselves up, but cushioned as to not make it uncomfortable to sit in. A small firepit has been built in the middle of the tent, along with a couple fire extinguishers placed nearby, and the whole place is brightly lit in contrast to Burthen's typically chilly weather. Quite a few people, both Burthenians and offworlders, are mingling about here.`
```

> 			`	Thankfully the walk to the catering tent is far shorter than the walk from your ship, but nevertheless you're exhausted by the time you drag your feet inside. A lot of effort has been put into this tent: the metal chairs are all sturdy to allow offworlders not used to the gravity to pull themselves up, but cushioned as to not make it uncomfortable to sit in; a small firepit has been built in the middle of the tent, along with a couple fire extinguishers placed nearby; and the whole place is brightly lit in contrast to Burthen's typically chilly weather. Quite a few people, both Burthensiders and offworlders, are mingling about here.`

Reviewer: ```suggestion
			`	Thankfully the walk to the catering tent is far shorter than the walk from your ship, but nevertheless you're exhausted by the time you drag your feet inside. A lot of effort has been put into this tent: the metal chairs are all sturdy to allow offworlders not used to the gravity to pull themselves up, but cushioned as to not make it uncomfortable to sit in. A small firepit has been built in the middle of the tent, along with a couple fire extinguishers placed nearby, and the whole place is brightly lit in contrast to Burthen's typically chilly weather. Quite a few people, both Burthensiders and offworlders, are mingling about here.`
```

You could probably keep the second semicolon, but you can't keep both.

Author: I'll commit it because it does read a bit better, but I'm pretty sure semicolons can be used this way.

~~Last time I got burned for misreading and referencing a website on grammar.~~ https://www.merriam-webster.com/grammar/a-guide-to-using-semicolons

Reviewer: Well, you _can_ use semicolons that way, but no one wants to read a sentence with two of them in it, especially since there's only a couple hundred in the data files in total.

Author: but I love semicolons!

> 			`	There's a counter in the corner with two levels, apparently so that the Burthensider attending it can serve both offwolders and locals. Alongside the typical drinks you've seen in spaceport bars and some local to the southern Core, the counter also offers a local drink called the "Antigrav" that claims to "ease visitors from Burthen's natural pull", and comes in a specially made glass, all for 30 credits.`

Author: ```suggestion
			`	There's a counter in the corner with two levels, apparently so that the Burthenian attending it can serve both offwolders and locals. Alongside the typical drinks you've seen in spaceport bars and some local to the southern Core, the counter also offers a local drink called the "Antigrav" that claims to "ease visitors from Burthen's natural pull", and comes in a specially made glass, all for 30 credits.`
```

> 			`	There's a counter in the corner with two levels, apparently so that the Burthensider attending it can serve both offwolders and locals. Alongside the typical drinks you've seen in spaceport bars as well as some ones local to the southern Core, the counter also offers a local drink called the 'Antigrav' that claims to "ease visitors from Burthen's natural pull", and comes in a specially made glass, all for 30 credits.`

Reviewer: ```suggestion
			`	There's a counter in the corner with two levels, apparently so that the Burthensider attending it can serve both offwolders and locals. Alongside the typical drinks you've seen in spaceport bars and some local to the southern Core, the counter also offers a local drink called the "Antigrav" that claims to "ease visitors from Burthen's natural pull", and comes in a specially made glass, all for 30 credits.`
```

			branch "no credits"
				credits < 30
			choice
				`	(Order a drink from the counter.)`

> 				`	(Order the 'Antigrav' from the counter.)`

Reviewer: ```suggestion
				`	(Order the Antigrav from the counter.)`
```

There's no reason to use single quotes anywhere in here. Also, you probably don't need quotes at all after the first time you use the word Antigrav.

Author: Yeah, I get single and double quotes confused. ~~I'm still a grammar reviewer.~~

					goto antigrav
				`	(Don't order anything and leave the tent.)`
					goto "leave tent"

			action
				payment -2

> 			`	You hand over a couple credits for the drink and leave the tent, sipping with some difficulty as you go; the Burthensider at the counter advises you not to drink too much of it, and not to drop the cup "unless you want a munted foot for a few weeks."`

Reviewer: ```suggestion
			`	You hand over a couple credits for the drink and leave the tent, sipping with some difficulty as you go. The Burthensider at the counter advises you not to drink too much of it, and not to drop the cup "unless you want a munted foot for a few weeks."`
```

This sentence is a bit long and the semicolon provides no real benefit.

				goto "leave tent"

			label antigrav
			action

> 				payment -30

Reviewer 2: The Mug is 50 credits. This feels too cheap in comparison.

Author: Well, the Mug is meant to be 5 times the price of an average mug (10c), while this mug is meant to be a normal mug with a gimmick. Having a mug that's cheaper would emphasise how dear the other one is, no?

				outfit "Antigrav Glass"

> 			`	It turns out the glass the Antigrav comes with has a miniature repulsor installed so that you don't need to hold it while drinking it. The drink itself is a sort of highly carbonated soda that tastes like something in between lingonberry and raspberry, made from the native berries found in Westland, the planetside district the spaceport is located in. The Burthensider at the counter advises you not to drink too much of it at once and to not turn it off while it's levitating, "lest it falls on your foot and we have to treat your bone fractures."`

Author: ```suggestion
			`	It turns out the glass the Antigrav comes with has a miniature repulsor installed so that you don't need to hold it while drinking it. The drink itself is a highly carbonated soda that tastes like something in between lingonberry and raspberry, made from the native berries found in Westland, the planetary district the spaceport is located in. The Burthenian at the counter advises you not to drink too much of it at once and to not turn it off while it's levitating, "lest it falls on your foot and we have to treat your bone fractures."`
```

> 			`	It turns out the glass the Antigrav comes with has a miniature repulsor installed so that you don't need to hold it while drinking it. The drink itself is a sort of highly carbonated soda that tastes like something in between lingonberry and raspberry, made from the native berries found in Westland, the planetside district the spaceport is located in. The Burthensider at the counter advises you not to drink too much of it at once and to not turn it off while it's levitating, "lest it falls on your foot and we have to treat your bone fractures."`

Reviewer 2: "Planetary" is already implicit by the fact that the player is on a planet.

> 			`	It turns out the glass the 'Antigrav' comes with has a miniature repulsor installed so that you don't need to hold it while drinking it; the 'Antigrav' itself is a sort of highly carbonated soda that tastes like something in between lingonberry and raspberry, made from the native berries found in 'Westland', the planetside country the spaceport is located in. The Burthensider at the counter advises you not to drink too much of it at once and to not turn it off while it's levitating, "lest it falls on your foot and we have to treat your bone fractures."`

Reviewer: ```suggestion
			`	It turns out the glass the Antigrav comes with has a miniature repulsor installed so that you don't need to hold it while drinking it. The drink itself is a sort of highly carbonated soda that tastes like something in between lingonberry and raspberry, made from the native berries found in Westland, the planetside district the spaceport is located in. The Burthensider at the counter advises you not to drink too much of it at once and to not turn it off while it's levitating, "lest it falls on your foot and we have to treat your bone fractures."`
```

Just because you _can_ technically write out a full paragraph in one sentence using semicolons doesn't mean it's a good idea. The word "country" implies either autonomy or rurality, at least to me, neither of which seem appropriate in this context.

Is there a specific type of "native berry" grown in Westland?

Author: 1. Err, I did write this one funny. It was almost a stream of consciousness thing, too bad I have to follow the rules here. "Planetary district" is probably the right word to use as well, I forgot about that one.

2. Yes.

				goto "leave tent"

			label "no credits"
			`	Unfortunately, it looks like all the drinks are too pricy for you.`
			choice
				`	(Don't order anything and leave the tent.)`

			label "leave tent"

> 			`	The crowd appears to be a bit more active when you return outside, and over the heads of the locals you see on a large, portable screen that the leading triathlete is a particularly stocky Burthensider, wearing a white helmet and some sort of tight, dark sportswear.`

Author: ```suggestion
			`	The crowd is a bit more active when you return outside, and over the heads of the locals you see on a large, portable screen that the leading triathlete is a particularly stocky Burthenian, wearing a white helmet and tight, dark sportswear.`
```

				goto "after tent"

			label outside

> 			`	Staying outside in the oppressive gravity and the chilly temperature, you move over to the sidelines and watch the large, portable screen that is broadcasting the leading competitors from above. It appears that competitors are divided into Burthensiders and offworlders, with Burthensiders wearing white helmets and offworlders wearing red helmets. It's nearing the end of the whole event; the competitor in front is named Abernathy, a stocky Burthensider apparently doing his eleventh Burthen Triathlon. Meanwhile, the leading offworlder is five hundred meters behind him, a competitor from Martini named Caldarone. Apparently, they've been at it for over 11 hours, beginning with a 1.5 kilometer swim, then a 40 kilometer bicycle ride, and finally a 10 kilometer run.`

Author: ```suggestion
			`	Staying outside in the oppressive gravity and the chilly temperature, you move over to the sidelines and watch the large, portable screen that is broadcasting the leading competitors from above. It appears that competitors are divided into Burthenians and offworlders, with Burthenians wearing white helmets and offworlders wearing red helmets. It's nearing the end of the whole event; the competitor in front is named Abernathy, a stocky Burthenian apparently doing his eleventh Burthen Triathlon. Meanwhile, the leading offworlder is five hundred meters behind him, a competitor from Martini named Caldarone. Apparently, they've been at it for over 11 hours, beginning with a 1.5 kilometer swim, then a 40 kilometer bicycle ride, and finally a 10 kilometer run.`
```

> 			`	After a couple minutes, Abernathy emerges from behind a bend, camera drones chasing him. Cheers, whoops, and rather overly ethusiastic hollers erupt from the crowd, raising enormously as he dashes through the finishing straight, the drones now circling around him. Just over the crowd a loudspeaker announces, "and here comes the winner of the Syndicate Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight Samuel Abernathy!"`

Author: I left a lot of commas out so that the pace would be faster.

> 			`	After a couple minutes, Abernathy emerges from behind a bend, camera drones chasing him. Cheers, whoops, and rather overly ethusiastic hollers erupt from the crowd, raising enormously as he dashes through the finishing straight, the drones now circling around him. Just over the crowd a loudspeaker announces, "and here comes the winner of the Syndicate Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight Samuel Abernathy!"`

Author: ```suggestion
			`	After a couple minutes, Abernathy emerges from behind a bend, camera drones chasing him. Cheers, whoops, and rather overly ethusiastic hollers erupt from the crowd, raising enormously as he dashes through the finishing straight, the drones now circling around him. Just over the crowd a loudspeaker announces, "And here comes the winner of the Syndicate Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight Samuel Abernathy!"`
```

> 			`	After a couple minutes, Abernathy emerges from behind a bend, camera drones chasing him. Cheers, whoops, and rather overly ethusiastic hollers erupt from the crowd, raising enormously as he dashes through the finishing straight, the drones now circling around him. Just over the crowd a loudspeaker announces, "and here comes the winner of the Syndicated Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight Samuel Abernathy!"`

Reviewer: ```suggestion
			`	After a couple minutes, Abernathy emerges from behind a bend, camera drones chasing him. Cheers, whoops, and rather overly ethusiastic hollers erupt from the crowd, raising enormously as he dashes through the finishing straight, the drones now circling around him. Just over the crowd a loudspeaker announces, "and here comes the winner of the Syndicated Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight, Samuel Abernathy!"`
```

I'm not sure how to improve this, but I doubt Abernathy's number was `fifty-eight Samuel Abernathy`.

Author: No, it's more that the announcer didn't pause between 'eight' and 'Samuel'. His number is 58.

				goto end

			label "after tent"

> 			`	As he nears the finish line the Burthensiders' enthusiastic cheers, whoops, and hollers erupt from the crowd, and the camera drones, their repulsors apparently now calibrated for Burthen's gravity, circle around the finishing straight. Just over the crowd a loudspeaker announces, "and here comes the winner of the Syndicate Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight, Samuel Abernathy!"`

Reviewer 2: ```suggestion
			`	As he nears the finish line the Burthensiders' enthusiastic cheers, whoops, and hollers erupt from the crowd, and the camera drones, their repulsors now calibrated for Burthen's gravity, circle around the finishing straight. Just over the crowd, a loudspeaker announces, "And here comes the winner of the Syndicate Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight, Samuel Abernathy!"`
```

Reviewer 3: I agree with removing 'apparently' but not the other two changes. The comma only limits the flow during what is meant to be a tense moment, and the 'a' should be capitalised if it starts a new sentence, but this one doesn't.

Reviewer 2: What sentence is the loudspeaker continuing, though?

Reviewer 3: The quote describing what the loudspeaker is saying continues the narration. '...Just over the crowd a loudspeaker announces, "and...' You don't need to join the second part of the quote to the first, if that makes any sense.

Reviewer 2: Unless it is continuing previous dialogue in the same sentence, the first word of dialogue is always capitalised.

Author: ```suggestion
			`	As he nears the finish line the Burthensiders' enthusiastic cheers, whoops, and hollers erupt from the crowd, and the camera drones, their repulsors now calibrated for Burthen's gravity, circle around the finishing straight. Just over the crowd a loudspeaker announces, "And here comes the winner of the Syndicate Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight, Samuel Abernathy!"`
```

> 			`	As he nears the finishline the Burthensiders' ethusiastic cheers, whoops, and hollers erupt from the crowd, and the camera drones, their repulsors apparently now calibrated for Burthen's gravity, circle around the finishing straight. Just over the crowd a loudspeaker announces, "and here comes the winner of the Syndicated Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight Samuel Abernathy!"`

Reviewer: ```suggestion
			`	As he nears the finish line the Burthensiders' enthusiastic cheers, whoops, and hollers erupt from the crowd, and the camera drones, their repulsors apparently now calibrated for Burthen's gravity, circle around the finishing straight. Just over the crowd a loudspeaker announces, "and here comes the winner of the Syndicated Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight, Samuel Abernathy!"`
```

Same problem here, and "finish line" is two words.

			label "end"

> 			`	The Burthensider slows down into the finish line and seizes the tape, making a yelp of simultaneous joy and relief, and holds the tape up before everybody in the crowd and at the cameras pointed at him before accepting a bottle of water from someone on the sidelines.`

Reviewer: Would a professional athlete slow down before the finish line? Even with a lead like his, I feel like most athletes would push themselves right through the end.

Author: Maybe not; I watched a video of the ending of the Coast to Coast, a competition held in NZ, and that was my impression of what happened. But I could change this.

Reviewer: It probably also depends on the person, so it's okay as is. Maybe if the whole point is to beat the offworlders (and not set any records) it makes sense.

> 			`	The Burthensider slows down into the finish line and seizes the tape, making a yelp of simultaneous joy and relief, and holds the tape up before everybody in the crowd and at the cameras pointed at him before accepting a bottle of water from someone on the sidelines.`

Author: How about this?
```suggestion
			`	The Burthensider doesn't slow down until he's within an arm's length of the finishing line; as he crosses it he seizes the tape, making a yelp of simultaneous joy and relief. He holds the tape up before everybody in the crowd and at the cameras pointed at him, taking in the attention (and catching his breath) before accepting a bottle of water from someone on the sidelines.`
```

Reviewer: This one looks good. You don't technically need that semicolon either, but it works.

> 			`	The Burthensider doesn't slow down until he's within an arm's length of the finishing line; as he crosses it he seizes the tape, making a yelp of simultaneous joy and relief. He holds the tape up before everybody in the crowd and at the cameras pointed at him, taking in the attention (and catching his breath) before accepting a bottle of water from someone on the sidelines.`

Reviewer 2: ```suggestion
			`	Abernathy doesn't slow down until he's within an arm's length of the finishing line; as he crosses it he seizes the tape, making a yelp of simultaneous joy and relief. He holds the tape up before everybody in the crowd and at the cameras pointed at him, taking in the attention (and catching his breath) before accepting a bottle of water from someone on the sidelines.`
```

			label end
			`	You say your goodbyes as you board your ships. Although you are both captains, the trajectory of your lives have taken very different courses.`
				decline

mission "Burthen Triathlon"
	minor
	source
		planet "Burthen"
	to offer
		random < 1
	on offer
		conversation
			`Burthen's planetary spaceport is not known to be well-trafficked because of its planet's high gravity, but today it seems to have a lot of visitors. Many of the usually empty landing pads are now occupied by shuttlecraft, and as you step out onto the spaceport's rather untarnished apron, you notice there's a sizable crowd at the edge of the spaceport, some of them obviously from offworld judging by their height. There's even a few media crews standing by, although it looks like a couple of their camera drones are struggling in Burthen's gravity.`
			choice
				`	(See what's going on.)`
				`	(Avoid the commotion and tend to your own business.)`
					decline

> 			`	Every step takes significantly more effort than you're used to, but you eventually reach the crowd. From the signs that have been put up on the way it looks like the 'Burthen Triathlon' is taking place today, and the spaceport has been designated the finishing point. A couple of especially sturdy-looking marquees have been set up; the smaller one appears to be the administration tent, currently blocked off to non-competitors and staff, while the other one is far larger and is catering food and drinks for the crowd nearby.`

Reviewer 4: > currently blocked off to non-competitors and staff

So even staff can't get into their own tent?

Reviewer 3: How about, “…currently only accessible to competitors and staff”?

> 			`	Every step takes significantly more effort than you're used to, but you eventually reach the crowd. From the signs that have been put up on the way it looks like the 'Burthen Triathlon' is taking place today, and the spaceport has been designated the finishing point. A couple of especially sturdy-looking marquees have been set up; the smaller one appears to be the administration tent, currently blocked off to non-competitors and staff, while the other one is far larger and is catering food and drinks for the crowd nearby.`

Author: ```suggestion
			`	Every step takes significantly more effort than you're used to, but you eventually reach the crowd. From the signs that have been put up on the way it looks like the 'Burthen Triathlon' is taking place today, and the spaceport has been designated the finishing point. A couple of especially sturdy-looking marquees have been set up; the smaller one appears to be the administration tent, currently only accessible to event staff, while the other one is far larger and is catering food and drinks for the crowd nearby.`
```

> 			`	Every step takes significantly more effort than you're used to, but you eventually reach the crowd. From the signs that have been put up on the way, it looks like the 'Burthen Triathlon' is taking place today, and the spaceport has been designated the finishing point. A couple of especially sturdy-looking marquees have been set up; the smaller one appears to be the administration tent, currently blocked off to non-competitors and staff, while the other one is far larger and is catering food and drinks for the crowd nearby.`

Reviewer 3: ```suggestion
			`	Every step takes significantly more effort than you're used to, but you eventually reach the crowd. From the signs that have been put up on the way it looks like the 'Burthen Triathlon' is taking place today, and the spaceport has been designated the finishing point. A couple of especially sturdy-looking marquees have been set up; the smaller one appears to be the administration tent, currently blocked off to non-competitors and staff, while the other one is far larger and is catering food and drinks for the crowd nearby.`
```

			`	There's a counter in the corner with two levels, apparently so that the Burthensider attending it can serve both offwolders and locals. Alongside the typical drinks you've seen in spaceport bars as well as some ones local to the southern Core, the counter also offers a local drink called the 'Antigrav' that claims to "ease visitors from Burthen's natural pull", and comes in a specially made glass, all for 30 credits.`
			choice
				`	(Order a drink from the counter.)`
				`	(Order the 'Antigrav' from the counter.)`
					goto antigrav
				`	(Don't order anything and leave the tent.)`
					goto "leave tent"
			action
				payment -2
			`	You hand over a couple credits for the drink and leave the tent, sipping with some difficulty as you go; the Burthensider at the counter advises you not to drink too much of it, and not to drop it "lest it falls on your foot and we have to treat your bone fractures."`
				goto "leave tent"

			label antigrav
			action
				payment -30
				outfit "Antigrav Glass"

> 			`	It turns out the glass the 'Antigrav' comes with has a miniature repulsor installed so that you don't need to hold it while drinking it; the 'Antigrav' itself is a sort of highly carbonated soda that tastes in between lingonberry and raspberry, made from the native berries found in 'Westland', the planetside country the spaceport is located in. The Burthensider at the counter advises you not to drink too much of it at once and to not turn it off if it's levitating, "lest it falls on your foot and we have to treat your bone fractures."`

Author: ```suggestion
			`	It turns out the glass the 'Antigrav' comes with has a miniature repulsor installed so that you don't need to hold it while drinking it; the 'Antigrav' itself is a sort of highly carbonated soda that tastes like somthing in between lingonberry and raspberry, made from the native berries found in 'Westland', the planetside country the spaceport is located in. The Burthensider at the counter advises you not to drink too much of it at once and to not turn it off while it's levitating, "lest it falls on your foot and we have to treat your bone fractures."`
```

			label "leave tent"

> 			`	The crowd appears to be a bit more active when you return outside, and over the heads of the locals you see on a large, portable screen that the leading triathlete is a particularly stocky Burthensider, wearing a white helmet and wearing some sort of dark tight sportswear.`

Reviewer 3: ```suggestion
			`	The crowd appears to be a bit more active when you return outside, and over the heads of the locals you see on a large, portable screen that the leading triathlete is a particularly stocky Burthensider, wearing a white helmet and some sort of tight, dark sportswear.`
```

				goto "after tent"

			label outside
			`	Staying outside in the oppressive gravity and the chilly temperature, you move over to the sidelines and watch the large, portable screen that is broadcasting the leading competitors from above. It appears that competitors are divided into Burthensiders and offworlders, with Burthensiders wearing white helmets and offworlders wearing red helmets. It's nearing the end of the whole event; the competitor in front is named Abernathy, a stocky Burthensider apparently doing his eleventh Burthen Triathlon. Meanwhile, the leading offworlder is five hundred meters behind him, a competitor from Martini named Caldarone. Apparently, they've been at it for over 11 hours, beginning with a 1.5 kilometer swim, then a 40 kilometer bicycle ride, and finally a 10 kilometer run.`

> 			`	After a couple minutes, Abernathy emerges from behind a bend, camera drones chasing him. Cheers, whoops, and rather ethusiastic hollers erupt from the crowd, raising enormously as he dashes through the finishing straight, the drones now circling around him, and just over the crowd a loudspeaker announces, "and here comes the winner of the Syndicated Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight Samuel Abernathy!"`

Author: ```suggestion
			`	After a couple minutes, Abernathy emerges from behind a bend, camera drones chasing him. Cheers, whoops, and rather overly ethusiastic hollers erupt from the crowd, raising enormously as he dashes through the finishing straight, the drones now circling around him. Just over the crowd a loudspeaker announces, "and here comes the winner of the Syndicated Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight Samuel Abernathy!"`
```

				goto end

			label "after tent"
			`	As he nears the finishline the Burthensiders' ethusiastic cheers, whoops, and hollers erupt from the crowd, and the camera drones, their repulsors apparently now calibrated for Burthen's gravity, circle around the finishing straight. Just over the crowd a loudspeaker announces, "and here comes the winner of the Syndicated Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight Samuel Abernathy!"`

			label "end"
			`	The Burthensider slows down into the finish line and seizes the tape, making a yelp of simultaneous joy and relief, and holds the tape up before everybody in the crowd and at the cameras pointed at him before accepting a bottle of water from someone on the sidelines.`
				decline

outfit "Antigrav Glass"
	category "Hand to Hand"
	plural "Antigrav Glasses"
	cost 25
	thumbnail "outfit/unknown"
	"capture attack" .2
	"unplunderable" 1

> 	description "The miniature repulsor installed on the bottom makes this glass a bit heavier than other cups, so if the time ever comes that you need to knock someone else out with a drinking vessel, you guess this one will probably be more effective than anything else you've got."

Reviewer 3: ```suggestion
	description "When it is not activated, the miniature repulsor installed on the bottom makes this glass a bit heavier than other cups, so if the time ever comes that you need to knock someone else out with a drinking vessel, you guess this one will probably be more effective than anything else you've got."
```

				goto "leave tent"

			label antigrav
			action
				payment -30
				outfit "Antigrav Glass"
			`	It turns out the glass the Antigrav comes with has a miniature repulsor installed so that you don't need to hold it while drinking it. The drink itself is a highly carbonated soda that tastes like something in between lingonberry and raspberry, made from the native berries found in Westland, the district the spaceport is located in. The Burthenian at the counter advises you not to drink too much of it at once and to not turn it off while it's levitating, "lest it falls on your foot and we have to treat your bone fractures."`
				goto "leave tent"

			label "no credits"
			`	Unfortunately, it looks like all the drinks are too pricy for you.`
			choice
				`	(Leave the tent.)`

			label "leave tent"
			`	The crowd is a bit more active when you return outside, and over the heads of the locals you see on a large, portable screen that the leading triathlete is a particularly stocky Burthenian, wearing a white helmet and tight, dark sportswear.`
				goto "after tent"

			label outside

> 			`	Staying outside in the oppressive gravity and the chilly temperature, you move over to the sidelines and watch the large, portable screen that is broadcasting the leading competitors from above. It appears that competitors are divided into Burthenians and offworlders, with Burthenians wearing white helmets and offworlders wearing red helmets. It's nearing the end of the whole event; the competitor in front is named Abernathy, a stocky Burthenian apparently doing his eleventh Burthen Triathlon. Meanwhile, the leading offworlder is five hundred meters behind him, a competitor from Martini named Caldarone. Apparently, they've been at it for over 11 hours, beginning with a 1.5 kilometer swim, then a 40 kilometer bicycle ride, and finally a 10 kilometer run.`

Reviewer: Is 11 hours realistic? Current Olympic (Earth) times for the same event and distances are about three hours. Even factoring in Burthen's heavier gravity, I would still expect the native-born Burthenian world champion to be in the three to five hour range.

Author: Perhaps it isn't.

Originally the competition was going to be more like NZ's Coast to Coast, which is 243km of running, kayaking and biking. The record for that is 10:34:37, according to Wikipedia.

> 			`	After a couple minutes, Abernathy emerges from behind a bend, camera drones chasing him. Cheers, whoops, and rather overly ethusiastic hollers erupt from the crowd, raising enormously as he dashes through the finishing straight, the drones now circling around him. Just over the crowd a loudspeaker announces, "And here comes the winner of the Syndicate Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight Samuel Abernathy!"`

Reviewer: I honestly still feel like there should be a comma here after fifty-eight.

Reviewer 5: I think a colon is what's needed:

Author: ```suggestion
			`	After a couple minutes, Abernathy emerges from behind a bend, camera drones chasing him. Cheers, whoops, and rather overly ethusiastic hollers erupt from the crowd, raising enormously as he dashes through the finishing straight, the drones now circling around him. Just over the crowd a loudspeaker announces, "And here comes the winner of the Syndicate Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight: Samuel Abernathy!"`
```

				goto end

			label "after tent"

> 			`	As he nears the finish line the Burthensiders' enthusiastic cheers, whoops, and hollers erupt from the crowd, and the camera drones, their repulsors now calibrated for Burthen's gravity, circle around the finishing straight. Just over the crowd a loudspeaker announces, "And here comes the winner of the Syndicate Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight, Samuel Abernathy!"`

Author: ```suggestion
			`	As he nears the finish line the Burthensiders' enthusiastic cheers, whoops, and hollers erupt from the crowd, and the camera drones, their repulsors now calibrated for Burthen's gravity, circle around the finishing straight. Just over the crowd a loudspeaker announces, "And here comes the winner of the Syndicate Athletics Burthen Triathlon, the world triathlete champion, competitor number fifty-eight: Samuel Abernathy!"`
```
